.github/workflows/nestbuildmatrix.yml

```diff
@@ -298,8 +311,9 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+
       - name: "Run black..."
-        uses: psf/black@bf7a16254ec96b084a6caf3d435ec18f0f245cc7   # 23.3.0
+        uses: psf/black@193ee766ca496871f93621d6b58d57a6564ff81b  # 23.7.0
```

Comment:
During our developer meeting we discussed that we should be careful when bumping versions due to security concerns.

I have looked at the changes in v23.7.0 compared to v23.3.0 and have not seen any changes that raise a security concern. This particular package is maintained by the Python Software Foundation, which I think we generally can trust have a process in place for preventing nefarious code entering a release.